app/Jobs/CleanupOrphanedPreviewContainersJob.php

```
@@ -0,0 +1,202 @@
+    private function removeContainer($container, Server $server): void
+    {
+        $containerName = data_get($container, 'Names');
+        $applicationId = $this->extractApplicationId($container);
+        $pullRequestId = $this->extractPullRequestId($container);
+
+        Log::info('CleanupOrphanedPreviewContainersJob - Removing orphaned container', [
+            'container' => $containerName,
+            'application_id' => $applicationId,
+            'pull_request_id' => $pullRequestId,
+            'server' => $server->name,
+        ]);
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\Log' in method 'removeContainer'. (undefined)

(StaticAccess)